src/models/event.ts

```diff
@@ -167,6 +177,15 @@ export default class Event extends RestfulModel {
       modelKey: 'reminderMinutes',
       jsonKey: 'reminder_minutes',
     }),
+    reminderMethod: Attributes.String({
+      modelKey: 'reminderMethod',
+      jsonKey: 'reminder_method',
+    }),
+    reminders: Attributes.Object({
+      modelKey: 'reminders',
+      jsonKey: 'reminders',
+      itemClass: EventReminder,
+    }),
```

Comment:
I think we shouldn't throw an error if the reminders object exists in an event, but make it read only so we don't serialize it and invoke an API error.

```suggestion
    reminders: Attributes.Object({
      modelKey: 'reminders',
      jsonKey: 'reminders',
      itemClass: EventReminder,
      readOnly: true
    }),
```